### PR TITLE
Bump Fuzz CI Echidna from 1.7.1 to 1.7.2

### PR DESCRIPTION
**Pull request by naszam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,13 +95,13 @@ Allows governance to schedule a point in the future to end the vest. Used for pl
 
 ### Install Echidna
 
-- Building using Nix
+- Building using Nix
```
Comment (brianmcmichael) [inline README.md:98]: nit on these extra spaces

**Review by brianmcmichael — APPROVED** [review]
Looks good. Extra spaces in the readme trigger my OCD, but aren't blocking.

File: README.md (hunk 1 above)
Comment (naszam) [reply README.md]: Done.

**Review by brianmcmichael — APPROVED** [review]
LGTM

**Pull request merged**